docket.mk

```diff
@@ -1,4 +1,4 @@
-%_import : %.csv
+%_import : %.csv data
```

Comment:
this could be the dependency for using the new data download recipe in the existing import script